ports/stm/peripherals/exti.c

```diff
@@ -128,7 +125,7 @@ void EXTI4_IRQHandler(void) {
 #endif
 void EXTI9_5_IRQHandler(void) {
     uint32_t pending = EXTI->PR;
-    for (uint i = 5; i <= 9; i++) {
```

Comment:
Unlike in 8-bit microcontrollers, code performance and density in CircuitPython are not usually improved by manually using 'narrow' types. In some circumstances (not here), it can evn add extra instructions to mask off the high bits of the 32-bit architectural register. A canonical example of this is when you have two quantities to add that you know will never exceed a signed 16-bit value:
```
int add(int a, int b) {
    return a + b;
}

int16_t add16(int16_t a, int16_t b) {
    return a + b;
}
```

`add16` includes an extra `sxth` instruction for sign extending the result. Using the standard width integer type is therefore shorter and faster.

Reply:
@jepler thanks for the explanation. I can revert this change back to circuitpython's 'uint' and push another commit.

I am thinking to look into other ports and try to "sanitize" the header files...by just include what is used and nothing more. One benefit could be reduced build times ( to be seen :))

Reply:
I will wait for this minor change before approving and merging.